haswell/lib_haswell_memory_bound.cpp

```diff
@@ -187,11 +322,16 @@ int haswell_membound_other_stall_reason(metric_id_t metric_id,
 {
     // Update the counter values at the current sample period
     update_values(metric_id, current_sample_time);
+    *out_value= 0;
 
+    // this metric will have to be derived after combining the sets of metrics
+    // in a data format outside of map
+#if 0
```

Comment:
If you don't need this metric then just remove it.

Reply:
I will remove it.